docs/authentication/social-connections/tiktok.mdx

```diff
@@ -42,4 +44,4 @@ Once your app is approved, simply copy the **Client Key** and **Client Secret**
 
 ![Obtaining the Client Key and Client Secret](/docs/images/authentication-providers/tiktok/12885db4568578459fb4b9beab31838c929cf9d5-1382x833.png)
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with TikTok is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with TikTok is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with TikTok is now configured for your instance.
```